### DISCUSSION TITLE
Freetube ¿Alfred workflow?

### DISCUSSION THREAD

**Discussion by help-moo** [FreeTubeApp/FreeTube#3039] [General]
Hi everyone!
I'm looking for a way to search for videos in Freetube using Alfred (Macbook M1), but i havent found any workflow available. Maybe some of you know about one or knows from a way to do it. I think its posible to make  script (could be an Applescript) that could use the search query from Alfred and input it at the search bar in the Freetube App but i'm new to the scripting world. I'll be happy to learn more and do the script myself. If you have any advice it'd would help me a lot. 

Thanks in advance

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@PikachuEXE do you know something about this

**Reply by PikachuEXE** [reply]
workflow? GitHub workflow?
I don't think it's about using an actual computer (coz just use the GUI for it right?)

**Reply by ChunkyProgrammer** [reply]
I assume they're talking about this: https://www.alfredapp.com

**Reply by PikachuEXE** [reply]
I have zero knowledge about that...

**Comment by ChunkyProgrammer** [comment]
Our team is unfamiliar with using Alfred Workflows. I would recommend contacting Alfred Support for help with workflows 🙂
